install/nightlies.md

```diff
@@ -19,9 +19,10 @@ The [usual instructions]({{ '/install/#download-a-pre-compiled-binary-distributi
 If you have access to LCG, as it is the case on LXPLUS, for example, ROOT nightlies can be obtained by sourcing the relevant scripts:
 
 ```
-source /cvmfs/sft.cern.ch/lcg/nightlies/dev3/<DAY>/lcgenv/*/<PLATFORM+Compiler>/lcgenv-env.sh
-source /cvmfs/sft.cern.ch/lcg/nightlies/dev3/<DAY>/ROOT/HEAD/<PLATFORM+Compiler>/bin/thisroot.sh
+source /cvmfs/sft.cern.ch/lcg/views/dev3python3/latest/<Platform+compiler>/setup.sh
```

Comment:
interesting, so there is:

* `/cvmfs/sft-nightlies.cern.ch/lcg/views/dev3python3` (or dev4)
* `/lcg/views/dev3python3/` (and friends)
* `/cvmfs/sft.cern.ch/lcg/nightlies/dev3/` (and friends)

I'm confused :D I agree in general views are better than non-views, @Axel-Naumann might know why this was not a link to a view in the first place

Reply:
First and second are identical I think. Third is what the view will set up for you, so you don't have to know about it.

I will admit, though, that before you would set up only compiler and ROOT. Now, you set up *everything*.
I would argue, though, that that's OK if you want to test a nightly. I certainly prefer the one-point setup, that you don't need a `*` in the path, that you don't have to choose the day.